requirements.txt

```diff
@@ -17,7 +17,8 @@ tqdm
 gwdatafind
 
 # Requirements for full pegasus env
-pegasus-wms >= 5.0.1
+pegasus-wms.api >= 5.0.3
+GitPython>1.0 # Required by pegasus-wms.api, but not declared as such
```

Comment:
Should we flag this upstream?

Reply:
Possibly ... First want to check if this actually is the case. I'm not sure if this is something like GitPython should have been installed alongside the main pegasus executables at the system level, but instead it can find it here.

Reply:
Ah, ok. Have you checked this is really needed for generating the workflow though? It used to be the case that GitPython was just needed for some of the pegasus exes but not the actual API. If that is still the case, then a CI issue might be different as GitPython should have been pulled in by the pegasus system install.

Reply:
When I didn't have this the pegasus-plan jobs were failing. So I think this probably isn't a requirement of pegasus-wms.api, but a requirement of pegasus-plan that isn't being pulled in. Maybe the right approach is to install GitPython alongside the pegasus install in the github workflows.